README.rst

```diff
@@ -358,9 +358,24 @@ TravisCI allows Selenium tests to be run with any number of browser/OS
 combinations and `captures video from running
 tests <https://saucelabs.com/open_sauce/user/ivan-c>`__.
 
-UI tests can also be run locally (after installing ``xvfb``) by passing
-Tox the virtual environment that corresponds to the UI tests (``ui``):
+UI tests can also be run locally (after installing ``xvfb`` and geckodriver) by passing
+Tox the virtual environment that corresponds to the UI tests (``ui``).
 
+Setup
+^^^^^
+- ``sudo apt-get install xvfb python-xvfbwrapper``
```

Comment:
Could you remove `python-xvfbwrapper` here and add the below config to tox.ini (in the `[ui]` section)? Ideally, python modules should not be installed system-wide

```
deps =
    xvfbwrapper
    {[testenv]deps}
```

Reply:
xvfbwrapper is already part of requirements.text, so I left it there. Added ```deps = {[testenv]deps}``` though